tools/serverpod_cli/lib/src/test_util/builders/class_definition_builder.dart

```diff
@@ -320,4 +325,48 @@ class ClassDefinitionBuilder {
     _isException = isException;
     return this;
   }
+
+  ClassDefinitionBuilder withChildClasses(
+    String className,
+    String fileName,
+    String fieldName,
+    String fieldType,
+    bool fieldNullable,
+  ) {
+    _childClasses = [
+      ResolvedInheritanceDefinition(
+        ClassDefinitionBuilder()
+            .withClassName(className)
+            .withFileName(fileName)
+            .withSimpleField(
+              fieldName,
+              fieldType,
+              nullable: fieldNullable,
+            )
+            .build(),
+      )
+    ];
+    return this;
+  }
+
+  ClassDefinitionBuilder withExtendsClass(
```

Comment:
Suggestion: Same for this interface, please take in a single ClassDefinition and wrap it inside of this method.